sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -3332,6 +3340,105 @@ public void setDefaultRecoveryThreshold(@Nullable Long defaultRecoveryThreshold)
     }
   }
 
+  public static final class Logs {
+
+    /** Whether Sentry Logs feature is enabled and Sentry.logger() usages are sent to Sentry. */
+    @ApiStatus.Experimental private boolean enable = false;
+
+    /**
+     * Configures the sample rate as a percentage of log items to be sent in the range of 0.0 to
+     * 1.0. if 1.0 is set it means that 100% of items are sent. If set to 0.1 only 10% of items will
+     * be sent. Items are picked randomly. Default is 1.0
+     */
+    @ApiStatus.Experimental private @Nullable Double sampleRate = 1.0;
```

Comment:
to be removed

Reply:
Removed in https://github.com/getsentry/sentry-java/pull/4389